src/test/java/teammates/test/cases/BaseTestCase.java

```diff
@@ -116,23 +116,35 @@ protected static void ignorePossibleException() {
         assertTrue(true);
     }
     
+    /**
+     * Invokes the method named {@code methodName} as defined in the {@code definingClass}.
+     * @param definingClass     the class which defines the method
+     * @param methodName
+     * @param parameterTypes    the parameter types of the method,
+     *                          which must be passed in the same order defined in the method
+     * @param invokingObject    the object which invokes the method, can be {@code null} if the method is static
+     * @param args              the arguments to be passed to the method invocation
+     */
+    protected static Object invokeMethod(Class<?> definingClass, String methodName, Class<?>[] parameterTypes,
+                                         Object invokingObject, Object[] args)
+            throws Exception {
```

Comment:
Indentation should be deeper. These are easier on the eye (the break is less jarring)
```java
protected static Object invokeMethod(Class<?> definingClass, String methodName, Class<?>[] parameterTypes,
                                     Object invokingObject, Object[] args)  throws Exception {
```

```java
protected static Object invokeMethod(Class<?> definingClass, String methodName,
                                     Class<?>[] parameterTypes, Object invokingObject, Object[] args)  
                                                                                      throws Exception {
```